srtcore/core.cpp

```diff
@@ -7698,12 +7706,13 @@ bool srt::CUDT::updateCC(ETransmissionEvent evt, const EventVariant arg)
         // - m_dPktSndPeriod
         // - m_dCWndSize
         m_tdSendInterval    = microseconds_from((int64_t)m_CongCtl->pktSndPeriod_us());
-        m_dCongestionWindow = m_CongCtl->cgWindowSize();
+        double cgwindow = m_CongCtl->cgWindowSize();
```

Comment:
```suggestion
        const double cgwindow = m_CongCtl->cgWindowSize();
```

Reply:
Right. This is only required as intermediate because it's also used in logs, and in logs it's displayed as double.